tests/Unit/Instrumentation/AwsSdk/AwsSdkInstrumentationTest.php

```diff
@@ -37,6 +43,68 @@ public function testInstrumentationInit()
         );
     }
 
+    public function testSetXrayPropagator()
+    {
+        $this->assertInstanceOf(
+            Propagator::class,
+            new Propagator()
+        );
+    }
+
+    public function testGetXrayPropagator()
+    {
+        $awsSdkInstrumentation = new AwsSdkInstrumentation();
+        $propagator = new Propagator();
+        $awsSdkInstrumentation->setPropagator($propagator);
+
+        $this->assertSame(
+            $awsSdkInstrumentation->getPropagator(),
+            $propagator
+        );
+    }
+
+    public function testSetTracerProvider()
+    {
+        $spanProcessor = new SimpleSpanProcessor(new ConsoleSpanExporter());
```

Comment:
The start of this and the next couple of tests is duplicated, so you could move it into a `setUp()` method, and make the variables you need to refer to into class variables instead of local.